cpp_linter/rest_api/__init__.py

```diff
@@ -15,10 +17,46 @@
 COMMENT_MARKER = "<!-- cpp linter action -->\n"
 
 
+class RateLimitHeaders:
+    """A collection of HTTP response header keys that describe a REST API's rate limits.
+    Each parameter corresponds to a instance attribute (see below)."""
+
+    def __init__(self, reset: str, remaining: str, retry: str) -> None:
+        #: The header key of the rate limit's reset time.
+        self.reset = reset
+        #: The header key of the rate limit's remaining attempts.
+        self.remaining = remaining
+        #: The header key of the rate limit's "backoff" time interval.
+        self.retry = retry
```

Comment:
This looks fine for docs, but I wish this would be a [named tuple](https://docs.python.org/3/library/collections.html#namedtuple-factory-function-for-tuples-with-named-fields) instead of a mutable object.